### ISSUE TITLE
make dev fails on Manjaro

### ISSUE THREAD

**Issue by PhantomLel** [Guake/guake#2124]
I am trying to set up the development env on Manjaro linux. I ran the bootstrap script for Arch which installed all of the packages with no issue. However, I run into an issue when running `make dev` which gives me the following error https://pastebin.com/d0eHU8uw

**Comment by Davidy22** [comment]
Looks like some kind of pipenv issue. Try updating pipenv to the latest version and try again, if this continues to not work you may need to send info on your setup to the pipenv tracker

**Comment by PhantomLel** [comment]
https://pastebin.com/F1cEh6K6

I'm not exactly sure what I changed but the error has now changed to this. I have literally tried everything.

**Comment by Davidy22** [comment]
Does a manual install of the latest version of pycodestyle, the offending package, help?

**Comment by PhantomLel** [comment]
Yes it gets rid of the error in `make dev` but still `make run` does not work. Here is the error.
https://pastebin.com/8ZL4RsQT

**Comment by Davidy22** [comment]
Oh, I never actually tried those two make options since I took up the mantle on Guake, usually I run `make` and `make install`. I can observe similar messages with dev and run, can probably put a patch togethor to get those working

**Comment by PhantomLel** [comment]
Oh, so you just reinstall every time you make a change? This has to be updated then: https://guake.readthedocs.io/en/stable/contributing/dev_env.html because that's where I was getting info on how to set up the dev environment.

**Comment by Davidy22** [comment]
Been occupied but I'll mark this with a good first issue and hope someone swings in for hacktoberfest

**Comment by techrajdeep** [comment]
I am looking for an issue for my first contribution . Can you help me here.

**Comment by Davidy22** [comment]
The installation instructions are in the file `docs/source/user/installing.rst` I think. Most of the discussion on the process has been talked over in this issue, but make sure you actually install guake yourself from source so that you know first-hand what the process is because that's pretty important if you're going to be writing instructions on how to do it

**Comment by techrajdeep** [comment]
@Davidy22 Could you please assign it to me.

**Comment by Davidy22** [comment]
Just make the pr and send it in, don't need to be assigned anything
